rdr_service/resource/generators/participant.py

```diff
@@ -389,6 +390,32 @@ def _prep_participant(self, p_id, ro_session):
             'date_of_death': deceased_date_of_death
         }
 
+        # Collect participant pairing history
+        pairing_history = None
+        query = ro_session.query(ParticipantHistory.lastModified, ParticipantHistory.hpoId, HPO.name.label('hpo'),
+                                   ParticipantHistory.organizationId, Organization.externalId.label('organization'),
+                                   ParticipantHistory.siteId, Site.googleGroup.label('site')). \
+                join(HPO, HPO.hpoId == ParticipantHistory.hpoId).\
+                join(Organization, Organization.organizationId == ParticipantHistory.organizationId).\
+                join(Site, Site.siteId == ParticipantHistory.siteId).\
```

Comment:
Should these be `LEFT OUTER JOINS` rather than the default `INNER JOINS`?

Reply:
Yes, they should be.  Good catch.